### PR TITLE
feat: APP-3026 - Update CreateDao process to use standard transaction flow

### PR DESCRIPTION
**Pull request by cgero-eth**
## Description

- Update CreateDao process to use new standard transaction flow

Task: [APP-3026](https://aragonassociation.atlassian.net/browse/APP-3026)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-3026]: https://aragonassociation.atlassian.net/browse/APP-3026?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/createDAO.tsx [hunk 1]
```diff
@@ -340,126 +339,124 @@ export const CreateDAO: React.FC = () => {
...
+              symbol: formMethods.getValues('tokenSymbol'),
+              token_address: formMethods.getValues('tokenAddress.address'),
+              multisigWallets: formMethods.getValues('multisigWallets'),
+            })
+          }
+        >
+          <SetupCommunity />
+        </Step>
+        <Step
+          // *** optionally leaving previous common.js pointers which now work correctly also ***
```
Comment (sepehr2github) [inline src/pages/createDAO.tsx]: Maybe we can get rid of these comments right?

File: src/containers/transactionDialog/transactionDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    ? errorButtonLabel
+    : isSuccess
+    ? successButton.label
+    : idleButtonLabel;
+
+  const buttonAction = isSuccess ? successButton.onClick : sendTransaction;
+
+  const blockExplorer = chains.find(({id}) => id === chain?.id)?.blockExplorers
+    ?.default.url;
+  const transactionLink = `${blockExplorer}/tx/${txHash}`;
```
Comment (sepehr2github) [inline src/containers/transactionDialog/transactionDialog.tsx:97]: We have several explorer URL generator, in future we can move all of these to a utility funciton

File: src/containers/goLive/index.tsx [hunk 3]
```diff
@@ -59,27 +55,27 @@ const GoLive: React.FC = () => {
   const {votingType} = getValues();
 
   return (
-    <Container>
+    <div className="space-y-10 md:mx-auto md:w-3/4">
```
Comment (sepehr2github) [inline src/containers/goLive/index.tsx:58]: I'm still a fan of styled components and not direct className, but whatever you prefer

File: src/containers/createDaoDialog/utils/createDaoUtils.ts [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+      case 'token': {
+        const plugin =
+          votingType === 'gasless'
+            ? this.getGasslessPlugin(formValues)
+            : this.getTokenVotingPlugin(formValues);
+        plugins.push(plugin);
+        break;
+      }
+      default:
+        throw new Error(
```
Comment (sepehr2github) [inline src/containers/createDaoDialog/utils/createDaoUtils.ts:98]: Just checking, We are logging these errors using Sentry, right?

**Review by sepehr2github — APPROVED** [review]
Thanks for the great work Ruggero, Left a few minor comments!

File: src/pages/createDAO.tsx (hunk 1 above)
Comment (cgero-eth) [reply src/pages/createDAO.tsx]: Yes 👍🏻

File: src/containers/transactionDialog/transactionDialog.tsx (hunk 2 above)
Comment (cgero-eth) [reply src/containers/transactionDialog/transactionDialog.tsx:97]: I agree, maybe we could introduce an utility to the ODS

File: src/containers/createDaoDialog/utils/createDaoUtils.ts (hunk 4 above)
Comment (cgero-eth) [reply src/containers/createDaoDialog/utils/createDaoUtils.ts:98]: Yes exactly, all errors thrown on the App are logged to Sentry

File: src/containers/goLive/index.tsx (hunk 3 above)
Comment (cgero-eth) [reply src/containers/goLive/index.tsx:58]: I see, let's discuss this topic on a tech talk to be aligned for the refactoring/rebuilding of the app 👌🏻

**Pull request merged**